api/tests/app/test_camera_metrics.py

```diff
@@ -0,0 +1,903 @@
+import datetime
+import pytest
+from freezegun import freeze_time
+import numpy as np
+# The line below is absolutely necessary. Fixtures are passed as arguments to test functions. That is why IDE could
+# not recognized them.
+from api.tests.utils.fixtures_tests import config_rollback_create_cameras, heatmap_simulation, config_rollback, \
+    reports_simulation
+
+
+# TODO: Some endpoint need to give a range of dates, try to send only one date.
+
+
+# pytest -v api/tests/app/test_camera_metrics.py::TestsGetHeatmap
+class TestsGetHeatmap:
+    """ Get Heatmap, GET /metrics/cameras/{camera_id}/heatmap """
+    """
+    Returns a heatmap image displaying the violations/detections detected by the camera <camera_id>.
+    """
+
+    def test_get_one_heatmap_properly(self, config_rollback_create_cameras, heatmap_simulation):
+        # Make the request
+        camera, camera_2, client, config_sample_path = config_rollback_create_cameras
+        camera_id = camera["id"]
+
+        response = client.get(f"/metrics/cameras/{camera_id}/heatmap?from_date=2020-09-19&to_date=2020-09-19")
+
+        # Get the heatmap
+        heatmap_path = f"/repo/data/processor/static/data/sources/{camera_id}/heatmaps/violations_heatmap_2020-09-19.npy"
```

Comment:
I think it is best to use `os.path.join` to create paths. Also, the prefix could be stored as a constant at the top of the file

Reply:
A prefix was added and os.path.join() was implemented.